locale/en.json

```diff
@@ -22,7 +22,21 @@
 		"global.social-pinterest": "Pinterest",
 		"global.social-email": "Email",
 		"global.opens-in-new-window": "(Opens in new window)",
-		"global.sponsored-content": "Sponsored Content"
+		"global.sponsored-content": "Sponsored Content",
+		"global.social-email-content": "Send an email to %{authorName}",
```

Comment:
are these keys uploaded to lokalise? I remember we went over lokalise in onboarding. https://arcpublishing.atlassian.net/wiki/spaces/TI/pages/2538275032/Lokalise+and+Theme+Blocks

Reply:
oh cool I see it's been uploaded now